platzky/plugin/notifier.py

```
@@ -0,0 +1,38 @@
+    The engine enforces final routing — plugins cannot bypass user-configured
+    topic restrictions.
+    """
+
+    accepted_topics: set[NotificationTopic]
+
+    @abstractmethod
+    def notify(
+        self,
+        message: str,
+        topic: NotificationTopic,
+        attachments: list[AttachmentProtocol] = [],
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)